packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/textinput/ReactEditText.java

```diff
@@ -556,6 +557,18 @@ public void setFontFamily(String fontFamily) {
     mTypefaceDirty = true;
   }
 
+  @Override
+  public boolean setFontVariationSettings(@Nullable String fontVariationSettings) {
+    if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+      if (!Objects.equals(fontVariationSettings, getFontVariationSettings())) {
+        mTypefaceDirty = true;
```

Comment:
Do we read font variation setting in `maybeUpdateTypeface()`? Places where multiple props influence typeface are I think where we are wanting to store members, and do this sort of invalidation.

If we only ever need this state on the underlying EditText, we could just use `setFontVariationSettings()` and `getFontVariationSettings()`.

Reply:
The problem is that if we want to use `getFontVariationSettings()` we need to do sdk check. I am trying to figure out what can be the best option here.